internal/services/shared/tasktypes/step.go

```diff
@@ -220,20 +219,19 @@ func StepRunCancelToTask(stepRun *dbsqlc.GetStepRunForEngineRow, reason string)
 	stepRunId := sqlchelpers.UUIDToStr(stepRun.SRID)
 	tenantId := sqlchelpers.UUIDToStr(stepRun.SRTenantId)
 
-	payload, _ := datautils.ToJSONMap(StepRunNotifyCancelTaskPayload{
-		WorkflowRunId:   sqlchelpers.UUIDToStr(stepRun.WorkflowRunId),
+	payload, _ := datautils.ToJSONMap(StepRunCancelTaskPayload{
 		StepRunId:       stepRunId,
 		CancelledReason: reason,
 		StepRetries:     &stepRun.StepRetries,
 		RetryCount:      &stepRun.SRRetryCount,
 	})
 
-	metadata, _ := datautils.ToJSONMap(StepRunNotifyCancelTaskMetadata{
+	metadata, _ := datautils.ToJSONMap(StepRunCancelTaskMetadata{
 		TenantId: tenantId,
 	})
 
 	return &msgqueue.Message{
-		ID:       "step-run-cancelled",
+		ID:       "step-run-cancel",
```

Comment:
`step-run-cancelled` string still exists in 
- "/internal/services/controllers/jobs/controller.go"
- "/internal/services/dispatcher/dispatcher.go"
- "/internal/services/dispatcher/server.go"

Reply:
Because it's still an event. `step-run-cancel` is to cancel the step, `step-run-cancelled` is a notif that the step was cancelled.